main.py

```diff
@@ -112,6 +112,12 @@ def get(self):
 
 def get_redirect_route(regex_route, handler, defaults=None):
     """Returns a route that redirects /foo/ to /foo.
+    Arg:
```

Comment:
Duplicate of #7649. Can you please pick another part of the same issue?

Reply:
@DubeySandeep  sure bhaiya i'll pick another part of this same issue